backend/src/main/java/be/informatievlaanderen/vsds/demonstrator/Application.java

```diff
@@ -3,6 +3,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
```

Comment:
Did these imports pop up while refactoring? Because they are not used so I think they can be removed